Assets/MixedRealityToolkit.Tests/PlayModeTests/SpherePointerTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System.Collections;
+using UnityEditor;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    // Tests to verify sphere pointer distances
+    public class SpherePointerTests
+    {
+        // Keeping this low by default so the test runs fast. Increase it to be able to see hand movements in the editor.
+        private const int numFramesPerMove = 3;
```

Comment:
We should do this for all tests, good idea.

Reply:
Yes, and preferably all tests should use a shared value (right now there are still a few tests using 30 or so steps). I'll make a separate PR for that.

Reply:
#5358